shells/backends/gcp_tf_backend/src/driver.py

```
@@ -62,10 +63,9 @@ def get_inventory(self, context):
             try:
                 gcp_service = self._can_conntect_to_gcp(context, logger)
                 if not gcp_service:
-                    self._raise_and_log(logger, "There was an issue accessing GCP, please check authentication credentials.")
-                # raise ValueError("Can't connect to GCP")
+                    logger.exception("There was an issue accessing GCP, please check authentication credentials.")
```

There is no "raise" in this case. You only write to log. Dont we need to raise an error here?